user-access-service/src/main/java/no/unit/nva/database/DatabaseSubService.java

```diff
@@ -49,6 +52,7 @@ protected static Item fetchItemForTable(Table table, DynamoEntryWithRangeKey req
     }
 
     protected static <T> InvalidEntryInternalException handleError(Failure<T> fail) {
+        logger.error("Error fetching user:", fail.getException());
```

Comment:
Missing either "{}" or s/,/+/

Reply:
I am not sure whether this is a PMD bug (PMD also complains about this). The signature of the function is 

```
    public void error(String msg, Throwable t);
```
And according to Baeldung (https://www.baeldung.com/slf4j-log-exceptions) this is a legit way to print the exception that is being thrown. I could set the exception as an argument, but I am not sure that the whole stack-trace would be printed in this case.

Reply:
Apologies, in my mind this was getMessage!